### ISSUE TITLE
Text fields incorrectly defined with length

### ISSUE THREAD

**Issue by maxxer** [mdbtools/mdbtools#133]
I'm importing an mdb database with text fields. `mdb-schema` creates them with length, causing inserts to later fail for data too long:

```sql
CREATE TABLE `table1`
 (
	`idtable1`			int,
	`TITOLO`			varchar (400),
	`DATA`			date,
	`MATERIALE`			text (255),
	`MAX_PARTECIPANTI`			int,
);
```
the column `MATERIALE` should be `text` without size specification

**Comment by evanmiller** [comment]
Hi, which backend are you using? MSSQL, MySQL, etc

**Comment by maxxer** [comment]
MySQL

**Comment by evanmiller** [comment]
Fix included in #162
